File: pkg/models/talent.go
package models

type Talent struct {
	Name string `json:"name,omitempty"`
	Slug string `json:"slug,omitempty"`
	ID   int    `json:"ID,omitempty"`
	//IsInherent bool   `json:"isInherent,omitempty"`
	//Skills     []Skill `json:"skills,omitempty"`
	//Traits     []Trait `json:"traits,omitempty"`
}

File: client/src/interfaces/talent.ts
import {SkillOfTalentData} from "./skill";
import {TraitOfTalentData} from "./trait";
import {MonsterWithTalentData} from "./monster";

export interface TalentData {
    name: string
    slug: string
    id: number
    skills: SkillOfTalentData[]
    traits: TraitOfTalentData[]
}

export interface TalentOfMonsterData {
    name: string
    slug: string
    id: number
    isInnate: boolean
    skills: SkillOfTalentData[]
    traits: TraitOfTalentData[]
}